pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorTest.java

```diff
@@ -105,13 +104,7 @@ void setup() throws Exception {
     @Override
     @AfterClass(timeOut = 30000)
     void shutdown() throws Exception {
-        ForkJoinPool.commonPool().execute(() -> {
-            try {
-                super.shutdown();
-            } catch (Exception e) {
-                e.printStackTrace();
-            }
-        });
```

Comment:
This is just reverting a previous workaround for the timeouts which was just moving the problem somewhere else. The shutdown should happen in same thread and complete in time.